subsys/bootloader/image/CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ add_custom_command(
   --provision-addr ${provision_addr}
   --public-key-files ${ALL_PUBLIC_KEY_FILES}
   --output ${PROVISION_HEX}
+  --num-counter-slots ${CONFIG_SB_NUM_MONOTONIC_COUNTERS}
```

Comment:
How do we stop the user from configuring more slots than what is available? On 91 I guess we only have a limited amount of OTP flash available.

Reply:
Added a max-size parameter to provision.py.